src/known-organizations/index.js

```diff
@@ -153,6 +154,13 @@ export const KnownOrganizations = {
         name: 'NuCypher DAO',
         template: null,
       },
+      {
+        address: '0x3fa400483487A489EC9b1dB29C4129063EEC4654',
+        domain: 'cryptokek.eth',
```

Comment:
Perhaps the domain should be set to kek.aragonid.eth ?

Reply:
Yes I was going to ask you exactly this :smile: We could do this for now and update it later, if you decide to make `cryptokek.eth` point to `0xe00F7B744AB8333d64ed940dd36ed9398D8eDBD2`.